package.json

```
@@ -1,21 +1,30 @@
 {
-  "name": "dblapi.js",
-  "version": "2.4.1",
-  "description": "An official module for interacting with the top.gg API",
-  "main": "index.js",
+  "name": "@top-gg/sdk",
+  "version": "3.0.0",
```

This is going to be a new library since we can't push on top of dblapi.js so it should begin with `0.1.0`
```suggestion
  "version": "0.1.0",
```